### PR TITLE
new back button and content changes

### PR DESCRIPTION
**Pull request by vibhabamba**
Sharper back button. Tried to add a class .back-button like the
.store-icon class so the resolution could be managed on retina devices.
This hasn’t worked. Need to follow up on this.

The rest is minor copy changes.

### CODE REVIEW

**Comment by zahanm** [comment]
Can you delete the `images/backbutton_old.png` and `images/ios-gallery/backbutton.png`? They aren't needed.

> Removed

File: styles/index.less [hunk 1]
```diff
@@ -234,6 +239,8 @@ a {
   margin-top: 30px;
 }
 
+
```
Comment (zahanm) [inline styles/index.less]: take out these two unecessary newlines please.

**Comment by zahanm** [comment]
Looks like `.back-icon` didn't work because you weren't running `grunt watch`, so the CSS wasn't generated.

> Still does not seem to be generating the CSS. 
> I have updated the icon in the image folder to be a 104 X 104
> The back-icon class specifies its width to be 52 px. So if the css generates later, then the code should work.

File: styles/index.less (hunk 1 above)
Comment (vibhabamba) [inline styles/index.less]: Removed

**Comment by zahanm** [comment]
You can run `grunt less` to run just the less compilation task.

**Comment by vibhabamba** [comment]
screen shot 2015-06-25 at 1 19 20 am

**Comment by vibhabamba** [comment]
screen shot 2015-06-25 at 4 13 11 pm

**Comment by vibhabamba** [comment]
screen shot 2015-06-25 at 4 14 25 pm

**Comment by zahanm** [comment]
`ctrl + c` to abort that currently running command and get control back.
(you'll see the `$` when you get control back)

then `grunt less`

> that did work

**Comment by zahanm** [comment]
Can you remove `images/ios-gallery/backbutton.png` too?

> Done

**Comment by zahanm** [comment]
Awesome. Looking great. Where did your screenshot go?

**Comment by vibhabamba** [comment]
you know i do see a big green button here saying 'Merge Pull request'

**Pull request merged**